app/assets/stylesheets/utilities/layout.scss

```diff
@@ -13,3 +13,11 @@
 .inline-block {
   display: inline-block;
 }
+
+.dashboard-colour {
+  background:#2b8cc4;
+}
+
+.dashboard-font-size {
+  font-size: 35px;
```

Comment:
Similarly to colour, it'd be good to use the govuk classes here. There's [override classes](https://design-system.service.gov.uk/styles/typography/#font-override-classes) if you can't find the right size from the standard list.